packages/react-components/react-motion-components-preview/stories/src/Rotate/RotateDefault.stories.tsx

```diff
@@ -0,0 +1,232 @@
+    fontSize: tokens.fontSizeBase200,
+    fontWeight: tokens.fontWeightSemibold,
+    color: tokens.colorNeutralForeground2,
+    marginBottom: tokens.spacingVerticalXS,
+    borderBottom: `1px solid ${tokens.colorNeutralStroke2}`,
+    paddingBottom: tokens.spacingVerticalXS,
+  },
+  toggleGroup: {
+    display: 'grid',
+    gridTemplateColumns: '1fr 1fr',
+    gap: tokens.spacingHorizontalM,
+  },
+  ctaButton: {
+    flex: 1,
+  },
+  field: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: tokens.spacingVerticalXS,
+  },
+  sliderField: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: tokens.spacingVerticalXS,
+  },
+  sliderWrapper: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: tokens.spacingHorizontalS,
+  },
+  sliderLabel: {
+    fontSize: tokens.fontSizeBase200,
+    fontWeight: tokens.fontWeightMedium,
+    color: tokens.colorNeutralForeground1,
+  },
+  valueDisplay: {
+    fontSize: tokens.fontSizeBase100,
+    color: tokens.colorNeutralForeground2,
+    fontFamily: tokens.fontFamilyMonospace,
+  },
+});
+
+const LoremIpsum = () => (
+  <>
+    {'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. '.repeat(
+      10,
+    )}
+  </>
+);
+
+export const Default = (props: React.ComponentProps<typeof Rotate>) => {
+  const classes = useClasses();
+  const [visible, setVisible] = React.useState<boolean>(false);
+  const [autoplay, setAutoplay] = React.useState<boolean>(false);
+  const [perspective, setPerspective] = React.useState<string>('1000px');
+  const [duration, setDuration] = React.useState<number>(motionTokens.durationUltraSlow); // 500ms
+  const [axis, setAxis] = React.useState<Axis3D>('y');
+  const [angle, setEnterAngle] = React.useState<number>(-90);
+
+  const perspectiveSliderId = useId();
+  const durationSliderId = useId();
+  const enterAngleSliderId = useId();
+
+  const perspectiveMin = 200;
+  const perspectiveMax = 2000;
+  const durationMin = motionTokens.durationUltraFast; // 50ms
+  const durationMax = motionTokens.durationUltraSlow * 2; // 1000ms
+  const angleMin = -180;
+  const angleMax = 180;
+
+  return (
+    <div className={classes.container} style={{ perspective }}>
+      <div className={classes.controls}>
+        {/* Animation Controls Section */}
+        <div className={classes.controlSection}>
+          <div className={classes.toggleGroup}>
+            <Button className={classes.ctaButton} appearance="primary" size="large" onClick={() => setVisible(v => !v)}>
+              {visible ? 'Hide' : 'Show'}
+            </Button>
+            <Field>
+              <Switch
+                label="Autoplay"
+                checked={autoplay}
+                onChange={() => {
+                  if (!autoplay) {
+                    setVisible(!visible);
+                  }
+                  return setAutoplay(v => !v);
+                }}
+              />
+            </Field>
```

Comment:
I suggest to remove `autoplay`:
- it's not used in other motions' examples
- it complicates components logic